src/Plugin.php

```diff
@@ -27,22 +27,30 @@ public function init()
             'cart' => \fostercommerce\klaviyoconnect\services\Cart::class,
         ]);
 
+        $settings = $this->getSettings();
+
         Event::on(Fields::class, Fields::EVENT_REGISTER_FIELD_TYPES, function (RegisterComponentTypesEvent $event) {
             $event->types[] = \fostercommerce\klaviyoconnect\fields\ListField::class;
             $event->types[] = \fostercommerce\klaviyoconnect\fields\ListsField::class;
         });
 
         Event::on(User::class, User::EVENT_AFTER_SAVE, function (Event $event) {
-            Plugin::getInstance()->track->onSaveUser($event);
+            if ($settings->trackSaveUser) {
```

Comment:
Not a big deal but the `if` condition should probably wrap the event listener, since the event will not be triggered if no listeners are registered.

Reply:
Until other things are added to the event listener I suppose.

Reply:
@lukeholder I second-guessed myself on that, but figured it probably wouldn't make a difference? I'm not sure the overhead (if any) of listening on an event. Updated though